.binder/environment.yml

```
@@ -11,3 +11,4 @@ dependencies:
   - pandas-flavor >=0.1.2
   - scipy
   - jupyter
+  - ctd
```

We probably want to run the latest dev version in our demos.

```suggestion
  - pip
  - pip:
    - ctd @ git+https://github.com/pyoceans/python-ctd@main
    
```